week-06-intro-ruby-on-rails/day-04/drill/readme.md

```diff
@@ -1 +1,111 @@
 # <img src="https://cloud.githubusercontent.com/assets/7833470/10899314/63829980-8188-11e5-8cdd-4ded5bcb6e36.png" height="60"> Ruby Bubble Sort
+
+Run the tests in the `spec` folder, and write your code in `bubble_sort.rb`.
```

Comment:
@ben-manning Can you move this whole section about running tests/exploring code down to the bottom where you intro the challenge? It doesn't make sense here b/c they haven't cloned anything to get started yet.